test/integration/test_fyle_admin.py

```diff
@@ -0,0 +1,73 @@
+import logging
+from test.common.utilities import dict_compare_keys
+
+logger = logging.getLogger(__name__)
+
+
+def test_get_accounting_exports(fyle, mock_data):
+  """
+  Test Fyle admin accounting exports Object
+  :param fyle: fyle sdk instance
+  :param mock_data: mock instance
+  :return: None
+  """
+  # Get a list of all Admin Accounting_exports in a paginated manner and add to a list
+  accounting_export = []
+  query_params = {
+    'order': 'created_at.desc'
+  }
+
+  accounting_exports_generator = fyle.v1beta.admin.accounting_exports.list_all(query_params=query_params)
+  mock_accounting_exports = mock_data.accounting_export.get()
+
+  for response in accounting_exports_generator:
+    if response.get('data'):
+      accounting_export.extend(response['data'])
+
+  assert dict_compare_keys(accounting_export[0], mock_accounting_exports[0]) == [], 'fyle.v1beta.admin.accounting_exports.list_all() has stuff that mock_data doesnt'
```

Comment:
handle null cases as well, do all these checks only when you have at least 1 `accounting_export`